src/pages/HomePage/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,34 @@
+import { CategoryCard } from '../CategoryCard';
+import './Categories.scss';
+
+export const Categories: React.FC = () => {
+  const phoneModelsAmount = 3;
```

Comment:
Looks like a magic numbers here. Let's imitate env variables, by declaring these limits outside the component scope in uppercase.